docs/source/extensions/datamodules.rst

```diff
@@ -432,3 +432,21 @@ You can of course use DataModules in plain PyTorch code as well.
 
 But overall, DataModules encourage reproducibility by allowing all details of a dataset to be specified in a unified
 structure.
+
+----------------
+
+Hyperparameters in DataModules
+------------------------------
+Like LightningModules, DataModules support hyperparameters with the same API.
+
+.. code-block:: python
+
+    import pytorch_lightning as pl
+
+
+    class CustomDataModule(pl.LightningDataModule):
+        def __init__(self, *args, **kwargs):
+            super().__init__()
+            self.save_hyperparameters()
+
+Refer to `save_hyperparameters` in :doc:`lightning module <../common/lightning_module>` for more details.
```

Comment:
```suggestion
Refer to :meth:`~pytorch_lightning.core.lightning.LightningModule.save_hyperparameters` for more details.
```
Can you check if linking to it directly would work?

Reply:
it only highlights unfortunately